x-pack/plugins/secops/public/components/page/network/network_dns_table/__snapshots__/is_ptr_included.test.tsx.snap

```diff
@@ -0,0 +1,8 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`NetworkTopNFlow Select direction rendering it renders the basic group button for uni-direction and bi-direction 1`] = `
```

Comment:
Looks like `renders the basic group button for uni-direction and bi-direction ` should be updated to `renders toggle for 'Include PTR Records'`. ++ again on us chatting about reviewing snapshots @XavierM :)